TrappingRainWater/main.cpp

```diff
@@ -0,0 +1,91 @@
+#include "main.h"
+
+#include <iostream>
+
+using namespace std;
+
+int Solution::trap(vector<int>& height) {
```

Comment:
I'll comment on the algorithm once I know what the task is and what the algorithm is. :)